test/favorites.test.js

```diff
@@ -0,0 +1,77 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const expect = chai.expect;
+chai.use(chaiHttp);
+
+const app = require('../index').app;
+const db = require('knex')(require('../knexfile')['development']);
+
+describe("Favorites Endpoints", () => {
+  before((done) => {
+    db.raw("TRUNCATE playlists_favorites restart identity;")
+    .then(() => db.raw("TRUNCATE playlists restart identity CASCADE;"))
+    .then(() => db.raw("TRUNCATE favorites restart identity CASCADE;"))
+    .then(() => done())
+    .catch(error => { throw error; });
+  });
+
+  before((done) => {
+    db.migrate.latest()
+    .then(() => done())
+    .catch(error => { throw error; });
+  });
+
+  before((done) => {
+    db.seed.run()
+    .then(() => done())
+    .catch(error => { throw error; });
+  });
+
+  it("GET /api/v1/favorites", (done) => {
```

Comment:
I still have trouble understanding done in this line. This all looks good though.